cupy_backends/cuda/libs/profiler.pyx

```diff
@@ -14,8 +14,12 @@ ELSE:
         cudaError_t cudaProfilerInitialize(const char *configFile,
                                            const char *outputFile,
                                            int outputMode)
-        cudaError_t cudaProfilerStart()
-        cudaError_t cudaProfilerStop()
+        IF CUPY_HIP_VERSION != 0:
```

Comment:
Inside the cpdef functions, can we use ``` runtime._is_hip_environment()``` instead of Cython env variables, to make it consistent across all the other files. 
Example: 
https://github.com/ROCmSoftwarePlatform/cupy/blob/hipify-cupy/cupy_backends/cuda/libs/curand.pyx#L114

Reply:
Changed it here: https://github.com/ROCmSoftwarePlatform/cupy/pull/26/commits/a3687ed8603b298584ab0e3e20642bb9714e0696